Sprint-3/slideshow/index.html

```diff
@@ -3,12 +3,21 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Sam Image Carousel</title>
     <script defer src="slideshow.js"></script>
+    <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <img id="carousel-img" src="./assets/cute-cat-a.png" alt="cat-pic" />
-    <button type="button" id="backward-btn">Backwards</button>
-    <button type="button" id="forward-btn">Forward</button>
+    <section id="images">
+      <img id="carousel-img" src="./assets/cute-cat-a.png" alt="cat-pic" />
+      <p id="image-count">Image 1 of 3</p>
+    </section>
+    <section id="buttons">
+      <button type="button" id="auto-forward">Auto Forward</button>
+      <button type="button" id="backward-btn">Backwards</button>
+      <button id="stop">Stop</button>
+      <button type="button" id="forward-btn">Forward</button>
+      <button type="button" id="auto-backward">Auto Backward</button>
```

Comment:
I found it a little surprising that "Auto Backwards" is in the "Forwards" direction end of the buttons, and "Auto Forwards" was in the "Backwards" direction end of the buttons. Is there a reason you ordered them like this? Would it perhaps be more clear if you swapped them?